### DISCUSSION TITLE
Welcome to vdm-vscode Discussions!

### DISCUSSION THREAD

**Discussion by jonaskrask** [overturetool/vdm-vscode#107] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nickbattle** [comment]
Hi. I'm Nick Battle, and I work with Jonas on the LSP Server that provides the language services in VDM VSCode. So I'll be dealing with any problems with the language engine (VDMJ) or the communication of language services to the Client.
